src/lib.rs

```diff
@@ -4,7 +4,7 @@ extern crate windows;
 
 pub mod helpers;
 
-#[allow(non_camel_case_types, non_snake_case, non_upper_case_globals, clippy::derivable_impls, clippy::missing_safety_doc, clippy::too_many_arguments)]
+#[allow(non_camel_case_types, non_snake_case, non_upper_case_globals, clippy::useless_transmute, clippy::extra_unused_lifetimes, clippy::derivable_impls, clippy::missing_safety_doc, clippy::too_many_arguments)]
 mod Microsoft;
 
 pub use Microsoft::Dia::*;
```

Comment:
Any reason we need this layer of modules?

Reply:
I see now, will simplify that, thanks!